src/components/ScreenWrapper.tsx

```diff
@@ -176,7 +177,9 @@ function ScreenWrapper(
     const {isBlurred, setIsBlurred} = useInputBlurContext();
 
     UNSTABLE_usePreventRemove(shouldReturnToOldDot, () => {
-        NativeModules.HybridAppModule?.closeReactNativeApp(false, false);
+        if (HybridAppModule.isHybridApp()) {
```

Comment:
Hmmmm do we need to check this? On standalone NewDot this method has no action so calling this without checking if we're on HybridApp should be fine. It will also simplify the code